huaweicloud/services/acceptance/cbr/data_source_huaweicloud_cbr_vaults_test.go

```diff
@@ -114,6 +116,11 @@ resource "huaweicloud_cbr_vault" "test" {
   enterprise_project_id = "0"
   backup_name_prefix    = "test-prefix-"
   is_multi_az           = true
+  auto_bind             = true
```

Comment:
Fields `auto_bind` and `bind_rules` are attribute references.
No need to configure in script.